perf/perf_mem.py.data/record_report.yaml

```diff
@@ -0,0 +1,73 @@
+record: !mux
+        event_load:
+                record_method: -e ldlat-loads
+        event__load_full:
+                record_method: --event ldlat-loads
+        event_store:
+                record_method: -e ldlat-stores
+        event__store_full:
+                record_method: --event ldlat-stores
+        all_kernel:
+                record_method: -K -v
+        all_kernel_full:
+                record_method: --all-kernel --event ldlat-stores
+        all_user:
+                record_method: -U -e ldlat-loads
+        all_user_full:
+                record_method: --all-user
+        input_data:
+                record_method: -i -e ldlat-stores
+        input_data_full:
+                record_method: --input -e ldlat-stores
+        force:
+                record_method: -f --event ldlat-stores
+        force_full:
+                record_method: --force -t load
+        type_load:
+                record_method: -t load -e ldlat-stores
+        type_store:
+                record_method: -t store --event ldlat-loads
+        type_load_full:
+                record_method: --type=load -C 0 -e ldlat-loads
+        type_store_full:
+                record_method: --type=store
+        cpu_verbose:
+                record_method: -C 0 --verbose
+        cpu_full:
+                record_method: --cpu=0
+        Physical_address:
+                record_method: -p
+        Physical_address_full:
+                record_method: --phys-data
+        data_page_size:
+                record_method: --data-page-size
+        event_store_load:
+                record_method: -e ldlat-stores -t load -f -i -v -C 0
```

Comment:
same here also, can change it to
`record_method: -e ldlat-stores -t load -f -v -C 0`